docs/streams/developer-guide/config-streams.html

```diff
@@ -281,7 +283,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
             <td colspan="2">The frequency in milliseconds with which to save the position (offsets in source topics) of tasks.</td>
             <td>30000 milliseconds (30 seconds)</td>
           </tr>
-          <tr class="row-odd"><td>default.deserialization.exception.handler</td>
+          <tr class="row-odd"><td>default.deserialization.exception.handler (Deprecated. Use deserialization.exception.handler instead.)</td>
```

Comment:
In this table, we should also insert a row for `deserialization.exception.handler`? Which can be basically a copy of this row?

Or do we think it would be redundant? For this case, we might want to rename this row to the new `deserialization.exception.handler`?

Not sure how to structure this nicely. Thoughts?

Reply:
yea it makes sense to add a new one, and in future we can delete this deprecated row totally.
Added new row.